ChebsNecromancy/Minions/SkeletonMinion.cs

```diff
@@ -587,11 +600,18 @@ public static void CountActiveSkeletonMinions()
             // when we kill off surplus, the oldest things are getting killed
             // not the newest things
             minionsFound = minionsFound.OrderByDescending((arg) => arg.Item1).ToList();
+            
+            var playerNecromancyLevel =
+                Player.m_localPlayer.GetSkillLevel(SkillManager.Instance.GetSkill(BasePlugin.NecromancySkillIdentifier).m_skill);
+            var bonusMinions = DraugrMinion.MinionLimitIncrementsEveryXLevels.Value > 0
```

Comment:
wrong minion type (draugr instead of skelly)

Reply:
Nice catch, thanks